storage/innobase/buf/buf0buf.cc

```diff
@@ -495,6 +495,9 @@ buf_get_total_stat(
 
 		tot_stat->n_pages_not_made_young +=
 			buf_stat->n_pages_not_made_young;
+
```

Comment:
Should this change also be propagated to 8.0?

Reply:
Yes, but I wanted 5.7 ready and reviewed first before making and testing 8.0 branch.  I also needed a branch prepared for a custom build.